javers-core/src/main/java/org/javers/core/metamodel/property/BeanBasedPropertyScanner.java

```diff
@@ -31,4 +31,12 @@ public BeanBasedPropertyScanner(AnnotationNamesProvider annotationNamesProvider)
         }
         return beanProperties;
     }
+
+    @Override
+    public Property scanSingleProperty(Class<?> managedClass, String propertyName) {
+        JaversMethod persistentGetter = ReflectionUtil.getPersistentGetter(managedClass, propertyName);
+
+
+        return null;
```

Comment:
a poważnie, to nie ma potrzeby powielać kodu i logiki w tej klasie, oraz wynikających z rego zmian niżej, czyli w ReflectionUtil, JaversFieldFactory i JaversMethodFactory

Możesz po prostu tu wywołać swoją własną metodę scan(clazz) i odfiltrować po nazwie
to samo dotyczy FieldBasedPropertyScannera.